README.md

```
@@ -35,74 +35,118 @@ Here's a simple example of how to create a client and list the sites in an organ
+```go
+package main
+
+import (
+    "log/slog"
+    "os"
+
+    "github.com/gregwight/mistclient"
+)
+
+func main() {
+	// Get a pointer to a slog.HandlerOptions with the TRACE level set.
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)